### PR TITLE
Update reference.md

### PR DESCRIPTION
**Pull request by sparky672**
The `onfocusout` option was improperly called `onblur` on this page.

### CODE REVIEW

File: pages/reference.md [hunk 1]
```diff
@@ -110,7 +110,8 @@ This used to work by adding `class="cancel"` to the input, this is now deprecate
 [Demo for the cancel button](https://jqueryvalidation.org/files/demo/errorcontainer-demo.html)
 
 # Validation event
-By default, forms are validated on submit, triggered by the user clicking the submit button or pressing enter when a form input is focused (option `onsubmit`). In addition, once a field was highlighted as being invalid, it is validated whenever the user types something in the field (option `onkeyup`). When the user enters something invalid into a valid field, it is also validated when the field loses focus (option `onblur`).
+
```
Comment (staabm) [inline pages/reference.md]: Please remove the added newline. Otherwise looks good, thx

Comment (sparky672) [inline pages/reference.md]: I think it's gone now.

**Comment by staabm** [comment]
Hmm the Pr is no longer mergable... let me fix this typo... thx for your contribution.

**Comment by staabm** [comment]
Fixed via https://github.com/jquery-validation/validation-content/commit/bda9cab7bf0d20b72650d48f446abc565891dcd3

**Pull request closed without merging**